.github/workflows/make-apk.yml

```diff
@@ -0,0 +1,30 @@
+name: Make Apk CI
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
This is great for power usage. To make things easier, do you know if there's a way to trigger the workflow directly from a PR page ?

Reply:
this will trigger the action manually and also automatically with a PR
```yaml
on:  
  workflow_dispatch:
  pull_request:
```

you can also do this
```yaml
on:  
  workflow_dispatch:
  push:
  pull_request:
```
so any pushed commit will trigger the action and generate a new debug_apk automatically